package.json

```diff
@@ -29,5 +29,8 @@
     "jshint-stylish": "^1.0.0",
     "load-grunt-tasks": "^3.1.0",
     "time-grunt": "^1.0.0"
+  },
+  "dependencies": {
+    "jquery": "^3.1.1"
```

Comment:
That would make this semantically incompatible with newer versions. What's your supported version range?